tests/tests/hrrr_control_2threads

```diff
@@ -0,0 +1,112 @@
+###############################################################################
+#
+#  HRRR control with 2 threads test
+#
+###############################################################################
+
+export TEST_DESCR="Compare HRRR with 2 threads results with control test"
+
+export CNTL_DIR=hrrr_control
+
+export LIST_FILES="sfcf000.nc \
+                   sfcf009.nc \
+                   sfcf012.nc \
+                   atmf000.nc \
+                   atmf009.nc \
+                   atmf012.nc \
+                   GFSFLX.GrbF00 \
+                   GFSFLX.GrbF09 \
+                   GFSFLX.GrbF12 \
+                   GFSPRS.GrbF00 \
+                   GFSPRS.GrbF09 \
+                   GFSPRS.GrbF12 \
+                   RESTART/coupler.res \
```

Comment:
Unless this test is going to be used to run subsequent restart test, which I think is not the case here, there's no need to save restart files in a baseline.

Reply:
This test does not have its own baseline. It uses hrrr_control. If you look at other decomp and 2threads variants, such as cpld_2threads_p8, you will see they do compare the restart files against their control.

```
export CNTL_DIR=hrrr_control
```